matflow/param_classes/load.py

```diff
@@ -871,6 +871,52 @@ def uniaxial_cyclic(
             out.extend(cycle_i)
         return [cls(**i)._remember_name_args(None, i) for i in out]
 
+    @classmethod
+    def from_npz_file(
+        cls,
+        npz_file_path,
+        idx,
+    ) -> list[Self]:
+        """
+        Read loading data from a numpy .npz file and format it appropriately.
+        Designed for running large arrays of simulations which each use loadcase of the
+        corresponding index.
+
+        Parameters
+        ----------
+        npz_file_path
+            A string of the filepath to the loadcase.npz file.
```

Comment:
Could you list which keys are required in this file, and what they represent here?